Files/Files.csproj

```diff
@@ -231,6 +232,8 @@
     <Compile Include="Helpers\XamlHelpers\DependencyObjectHelpers.cs" />
     <Compile Include="Helpers\XamlHelpers\SystemTypeToXaml.cs" />
     <Compile Include="Interacts\IStatusCenterActions.cs" />
+    <Compile Include="Helpers\SecondaryTileHelper.cs" />
+    <Compile Include="Helpers\XamlHelpers\SystemTypeToXaml.cs" />
```

Comment:
Not sure why but this file has been included twice (line 233)?

Reply:
Probably an issue from when I resolved merge conflicts.